CWE/common/countries/East Turkestan.txt

```diff
@@ -78,22 +78,34 @@ party = {
 		end_date = 2092.1.1
 }
 party = {
-		name = "Xianjiang Kuomintang" #RKMT branch in Xinjiang
+		name = "Uyghurstan People's Revolutionary Party" #Fictional
```

Comment:
This party name is more than 31 characters (including spaces)

Reply:
I've fixed it.